cirq/protocols/json_test_data/spec.py

```diff
@@ -69,6 +69,7 @@
         'ThreeQubitDiagonalGate',
         'Timestamp',
         'TwoQubitDiagonalGate',
+        'TwoQubitHeatmap',
```

Comment:
I think a `TwoQubitHeatmap` should not be serialized. We could move it to that category alongside with `Heatmap`.

Reply:
done